lecture_5/homework/spec/models/building_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+#
+# this tests was created due to TDD
+#
+#
+require 'rails_helper'
+require 'database_cleaner'
+
+DatabaseCleaner.clean_with :truncation
+
+RSpec.describe Building, type: :model do
+  describe 'creating new buildings' do
+    before(:each) do
+      @clan = create(:clan, id: 1, name: 'Klana')
```

Comment:
Zdecydowanie ładniej by to wyglądało przy wykorzystaniu `let!` lub `let` z wywołaniem w `before` :)